scripts/migrate_locales.js

```diff
@@ -0,0 +1,366 @@
+#!/usr/bin/env node
+/**
+ * A script to help integrators to migrate
+ * between different versions of Web SDK locale system.
+ *
+ * For more info, run:
+ *    $ migrate_locales --help
+ */
+
+'use strict' // eslint-disable-line strict
+
+const fs = require('fs')
+
+const COLORS = {
+  RESET: '\x1b[0m',
+  BLUE: '\x1b[34m',
+  GREEN: '\x1b[32m',
+  RED: '\x1b[31m',
+  YELLOW: '\x1b[33m',
+}
+const COMMAND = 'migrate_locales'
+const VERSION = 'v1.0.0'
+
+const VERSIONS = {
```

Comment:
This object might contain a lot of data just for the next version. I would assume that the data will increase every time we release a new version so I wonder if this object should leave in a separate file or we could even push it to our public S3 bucket and do a curl request to fetch it.

Reply:
This is out of scope, just something to think about

Reply:
Yeah, good idea about S3 bucket. Initially, I tent to use separate JSON files for versions, but it prevented the script's portability, so I decided to embed it directly in its code. I think for the very next version (`v1.0.0` maybe) we can still use this quite well, if we have any plans to support ongoing versions then it'll better be handled within next implementations.

Reply:
I agree. Could you create a ticket for that so we don't forget?

Reply:
[New ticket](https://jira.onfido.co.uk/browse/CX-5673) created!